applications/common/frontend/src/components/Navbar/navbar.js

```diff
@@ -35,6 +35,38 @@ document.addEventListener(window.KYANITE_ON_DOM_CONTENT_LOAD, () => {
 
         });
     });
+
+    // Get all navbar mega items
+    const $navbarMega = [...document.querySelectorAll('.navbar-end .navbar-item.has-dropdown')];
+    const desktopMQ = window.matchMedia('(min-width: 1024px)');
```

Comment:
Can we use a value from constants? https://github.com/websight-io/kyanite/blob/main/applications/common/frontend/src/helpers/breakpoints.ts#L20

Reply:
Done